Pasfmt.FormatEditor.pas

```diff
@@ -127,11 +79,7 @@ procedure TEditBufferFormatter.Format(Buffer: IOTAEditBuffer);
   CErrMsg = 'Format error';
```

Comment:
These two constants are unused.

Reply:
I would have expected sonar-delphi to report these as such. Is that a bug?

Reply:
SonarDelphi does, but DelphiLint doesn't because the dead code rules require full context to be useful.